docs/building-a-form/common-definitions.md

```diff
@@ -65,37 +65,59 @@ A common type-ahead widget that lets a user type in values and narrow down a lon
 
 ### Bank account
 
-The common electronic funds transfer (EFT) account information field that collects account type (checking or savings), bank account number, and bank routing number.
+The common electronic funds transfer (EFT) account information field that collects account type (checking or savings), bank account number, and bank routing number. Use these definitions as the uiSchema for the field that represents the account number.
 
-- File: [/src/js/definitions/bankAccount.js](../../src/js/definitions/bankAccount.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Object
+```js
+import { uiSchema as bankAccountUI }
+  from 'us-forms-system/lib/js/definitions/bankAccount'
+
+uiSchema: {
+  eftinfo: bankAccountUI;
+}
+```
+Source: [/src/js/definitions/bankAccount.js](../../src/js/definitions/bankAccount.js)
 
 ### Currency
 
-- File: [/src/js/definitions/currency.js](../../src/js/definitions/currency.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Function
+Formats and validates a US currency field. The display includes a leading `$` character. Call this exported function and pass it the label to be used on the field.
+
+```js
+import { uiSchema as currencyUI }
```

Comment:
I think this one, as well as most of the definitions, also have default exports, so might want to double check all the import statements in these code examples.